src/platforms/common/configuration/options.mdx

```diff
@@ -227,6 +227,24 @@ A path to an alternative CA bundle file in PEM-format.
 
 </ConfigKey>
 
+<ConfigKey name="frame-context-lines" supported={["node"]}>
+
+The number of context lines for each frame when loading a file.
+
+</ConfigKey>
+
+<ConfigKey name="initial-scope" supported={["javascript","node"]}>
+
+Data to be set to the initial scope.
+
+</ConfigKey>
+
+<ConfigKey name="max-value-length" supported={["node"]}>
+
+Maxium number of characters a single value can have before it will be truncated.
```

Comment:
Is this a duplicate of line 212? If so, suggest we simply add node to the list of `supported`? If not, there's a tiny typo :) 
```suggestion
Maximum number of characters a single value can have before it will be truncated.
```

Reply:
I think it's not the same with 212, at least the default values are different. Made minor updates. :)

Reply:
Thanks for following up.